src/build.d

```diff
@@ -241,19 +241,27 @@ version(Windows)
     // Build the msvc-dmc compiler wrapper
     auto buildMsvcDmc()
     {
+        enum targetName = "msvc-dmc";
+
         Dependency dependency = {
-            target: env["G"].buildPath("msvc-dmc").exeName,
-            sources: [`vcbuild\msvc-dmc`],
+            target: env["G"].buildPath(targetName).exeName,
+            sources: [`vcbuild\` ~ targetName],
+            name: "Building " ~ targetName,
+            command: [env["HOST_DMD_RUN"], "-of$@", "$<"]
         };
         return dependency;
     }
 
     // Build the msvc-lib linker wrapper
     auto buildMsvcLib()
     {
+        enum targetName = "msvc-lib";
+
         Dependency dependency = {
-            target: env["G"].buildPath("msvc-lib").exeName,
-            sources: [`vcbuild\msvc-lib`],
+            target: env["G"].buildPath(targetName).exeName,
+            sources: [`vcbuild\` ~ targetName],
+            name: "Building " ~ targetName,
+            command: [env["HOST_DMD_RUN"], "-of$@", "$<"]
```

Comment:
Sorry I thought that was the default. Should we set this as the default command?

Reply:
I didn't know there was a default command.  Please elaborate.